### ISSUE TITLE
Write prerequisites section for new packaging tutorials

### ISSUE THREAD

**Issue by ddbeck** [pypa/python-packaging-user-guide#196]
(For more information about this issue, please see #194).

We need a short section to follow the tutorial introduction (see #195) that makes sure the user has a working Python environment. This should line up with what's in [Install pip, setuptools, and wheel](https://packaging.python.org/en/latest/installing/#install-pip-setuptools-and-wheel), but using a more step-by-step approach (rather than the more self-directed approach of the other section). Additionally, it would be nice to have a step (or steps) that help readers verify that the tools are working.

When opening pull requests for this issue, please submit to the `develop` branch. If you have questions that concern contributing more generally, please use issue #194. Otherwise, feel free to comment with questions or feedback. Thanks!

**Comment by JoshuaAcosta** [comment]
Daniel, I can work on this section.

**Comment by ddbeck** [comment]
@JoshuaAcosta thanks!

**Comment by aryan-harsh** [comment]
@ddbeck  is this work still left. i would like to help

**Comment by pradyunsg** [comment]
As I understand, #396 is the same.

**Comment by ddbeck** [comment]
@aryan-harsh Yes, it seems like there's a lot left to do on this stuff. I don't have a lot of time for this project at the moment, but if you open PR, feel free to tag me and I'll see if I can at least give it a review. Thank you!

**Comment by deepa346** [comment]
I can work.

**Comment by mohdzahidabdullah** [comment]
> (For more information about this issue, please see #194).
> 
> We need a short section to follow the tutorial introduction (see #195) that makes sure the user has a working Python environment. This should line up with what's in [Install pip, setuptools, and wheel](https://packaging.python.org/en/latest/installing/#install-pip-setuptools-and-wheel), but using a more step-by-step approach (rather than the more self-directed approach of the other section). Additionally, it would be nice to have a step (or steps) that help readers verify that the tools are working.
> 
> When opening pull requests for this issue, please submit to the `develop` branch. If you have questions that concern contributing more generally, please use issue #194. Otherwise, feel free to comment with questions or feedback. Thanks!
